### PR TITLE
Spoke delegated

### PR DESCRIPTION
**Pull request by ujjwal-ibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMISVPC_dns_delegated_first
--- PASS: TestAccIBMISVPC_dns_delegated_first (295.69s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     298.237s
```

### CODE REVIEW

**Comment by astha-jain** [comment]
There are plenty of change I see in the PR .. unrelated to spoke delegate

File: website/docs/r/is_vpc.html.markdown [hunk 1]
```diff
@@ -83,6 +83,22 @@ resource "ibm_is_vpc" "example-system" {
 	}
 }
 
+// delegated type resolver
+
+resource "ibm_is_vpc" "example-delegated" {
+  // add a dependency on custom resolver for the hub vpc to have the resolver created first
```
Comment (ujjwal-ibm) [inline website/docs/r/is_vpc.html.markdown:89]: required : add a dependency on ibm dns custom resolver first

Comment (ujjwal-ibm) [reply website/docs/r/is_vpc.html.markdown]: changed

**Review by astha-jain — APPROVED** [review]

**Comment by ujjwal-ibm** [comment]
moving to upstream

**Pull request closed without merging**